File: sorting/selection_sort.py
def selection_sort(arr):
    for i in range(len(arr)):
        min_index = i
        for j in range(i + 1, len(arr)):
            if arr[min_index] > arr[j]:
                min_index = j
        arr[i], arr[min_index] = arr[min_index], arr[i]
    print(arr)


custom_list = [2, 1, 7, 6, 5, 3, 4, 9, 8]
selection_sort(custom_list)

File: sorting/selectionSort.js
function selectionSort(arr) {
  for (let i = 0; i < arr.length; i++) {
    let minIndex = i;
    for (let j = i + 1; j < arr.length; j++) {
      if (arr[minIndex] > arr[j]) {
        let temp = arr[minIndex];
        arr[minIndex] = arr[j];
        arr[j] = temp;
      }
    }
  }
  return arr;
}

const customList = [2, 1, 7, 6, 5, 3, 4, 9, 8];

console.log(selectionSort(customList));
console.log(customList);
